src/applications/personalization/view-dependents/tests/e2e/view-dependents.cypress.spec.js

```diff
@@ -11,26 +11,29 @@ const testAxe = () => {
 };
 
 const testHappyPath = () => {
-  testAxe();
   cy.intercept('GET', DEPENDENTS_ENDPOINT, mockDependents).as('mockDependents');
+  cy.visit(rootUrl);
+  testAxe();
   cy.findByText(/Your VA Dependents/).should('exist');
-  cy.findAllByText(/Date of birth/).should('have.length', 4);
+  cy.findAllByText(/Relationship/i, { selector: 'dfn' }).should(
```

Comment:
I'm sure this went through a11y review and I'm totally out of my depth, but it seems weird to have multiple `dfn`s for the same term on the same page. I also really lean toward doing `findByRole` since it ensures things work from an a11y perspective. It's possible to have weird code that redefines the role or disables the default role of an element. Like maybe if you had an anchor tag without an href, it _might_ no longer have a role of `link`. Long way of saying that I think you get a little extra insurance that things work nicely when you query by role. Just my two cents!

Reply:
Oh snap I was actually right about the `href` thing 
<img width="823" alt="Screen Shot 2021-03-04 at 3 09 38 PM 1" src="https://user-images.githubusercontent.com/20728956/110043192-d9f0bb00-7cfb-11eb-8f08-535e10d422f1.png"> via https://w3c.github.io/html-aria/#docconformance

Reply:
It's an interesting point you make about `dfn` tags. We might actually end up looking at refactoring this markup into something other than a definition list, since it was discovered that not all screen readers actually handle them correctly.

Reply:
Just discovered that `findByRole('term'...)` doesn't work unless you explicitly specify the `title` attribute of the `dfn`, which is kind of weird to me because it has an [implicit role](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dfn) of `term` 🤷  I agree though, after checking out `findByRole` it's way better than `findByText`

Reply:
>  it seems weird to have multiple dfns for the same term on the same page

@1Copenut - do you have any insight about @erikphansen's observation above? Curious is this is something to generally avoid.